howto/anbox/develop-platform.md

```diff
@@ -29,30 +29,32 @@ The build process creates a `platform_minimal.so` module in the `build` director
 
 ## Install the example platform
 
-[AMS](https://discourse.ubuntu.com/t/about-ams/24321) allows launching Anbox containers in a special [development mode](https://discourse.ubuntu.com/t/17763#dev-mode), which is helpful when developing, for example, addons or platforms. In development mode, the Anbox runtime does not terminate the container when it detects failures or other problems.
+[AMS](https://discourse.ubuntu.com/t/about-ams/24321) allows launching Anbox Cloud instances in a special [development mode](https://discourse.ubuntu.com/t/17763#dev-mode), which is helpful when developing, for example, addons or platforms. In development mode, the Anbox runtime does not terminate the instance when it detects failures or other problems.
 
 To try out the `minimal` platform, complete the following steps:
 
-1. Start a [raw container](https://discourse.ubuntu.com/t/managing-containers/17763#application-vs-raw) with development mode turned on:
+1. Start a [raw instance](https://discourse.ubuntu.com/t/17763#application-vs-raw) with development mode turned on:
 
         amc launch --raw --devmode --instance-type=a4.3
 
    If you chose the `nvidia` example, you must select an [instance type](https://discourse.ubuntu.com/t/application-manifest/24197#instance-type) that supports GPUs:
 
         amc launch --raw --devmode --instance-type=g4.3
 
-   The command prints out the ID of the container. Note down this ID; you will need it in the next step.
+    [note type="information" status="Note"]Use the `--vm` option to start a VM instance.[/note]
```

Comment:
Shouldn't we use the term "launch" instead of "start" here? "start" means starting an existing instance vs "launch" mean creating a new one and starting it immediately.